Neos.Neos/Resources/Private/Translations/ar/NodeTypes/Timable.xlf

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
-  <file original="" product-name="Neos.Neos" source-language="en" datatype="plaintext" target-language="ar">
-    <body>
-      <trans-unit id="properties._hiddenBeforeDateTime" xml:space="preserve">
-				<source>Hide before</source>
-			<target state="translated">إخفاء قبل</target></trans-unit>
-      <trans-unit id="properties._hiddenAfterDateTime" xml:space="preserve">
-				<source>Hide after</source>
-			<target state="translated">إخفاء بعد</target></trans-unit>
```

Comment:
we should consider migrating the exiting translation files to the new package.

Reply:
As the key and the meaning has changed that might not work.